src/main/java/edu/cornell/kfs/concur/batch/service/impl/ConcurRequestExtractFileServiceImpl.java

```diff
@@ -0,0 +1,112 @@
+package edu.cornell.kfs.concur.batch.service.impl;
+
+import org.apache.commons.io.FileUtils;
+import org.apache.commons.io.IOUtils;
+import org.apache.commons.lang.StringUtils;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.ArrayList;
+import java.util.List;
+
+import org.kuali.kfs.sys.batch.BatchInputFileType;
+import org.kuali.kfs.sys.batch.service.BatchInputFileService;
+import org.kuali.rice.core.api.util.type.KualiDecimal;
+
+import edu.cornell.kfs.concur.ConcurConstants;
+import edu.cornell.kfs.concur.batch.businessobject.ConcurRequestExtractFile;
+import edu.cornell.kfs.concur.batch.service.ConcurRequestExtractFileService;
+import edu.cornell.kfs.concur.batch.service.ConcurRequestExtractFileValidationService;
+
+public class ConcurRequestExtractFileServiceImpl implements ConcurRequestExtractFileService {
+    private static org.apache.log4j.Logger LOG = org.apache.log4j.Logger.getLogger(ConcurRequestExtractFileServiceImpl.class);
+    protected BatchInputFileType batchInputFileType;
+    protected BatchInputFileService batchInputFileService;
+    protected ConcurRequestExtractFileValidationService concurRequestExtractFileValidationService;
+
+    public List<String> getUnprocessedRequestExtractFiles() {
+        List<String> fileNamesToLoad = getBatchInputFileService().listInputFileNamesWithDoneFile(getBatchInputFileType());
+        return fileNamesToLoad;
+    }
+
+    public boolean processFile(String requestExtractFileName) {
+        boolean processingSuccess = true;
+        try {
+            Object parsedFile = loadFile(requestExtractFileName);
+            processingSuccess = requestExtractHeaderRowValidatesToFileContents(parsedFile);
+        } catch (RuntimeException re) {
+            LOG.error("Caught exception trying to load: " + requestExtractFileName, re);
+            processingSuccess = false;
+        }
+        return processingSuccess;
+    }
+
+    public boolean requestExtractHeaderRowValidatesToFileContents(Object parsedFile) {
```

Comment:
Moved method to validation class. Changed visibility of existing validation methods as they should no longer be public with this method now being in the validation class. Had to update tests as well due to this change.